source/accnt_mgmt.rst

```diff
@@ -2341,4 +2420,4 @@ To avoid this, you include the ``!`` which puts the macro into "quiet reference"
 
 ``Is your favorite color ?``
 
-Since customData can contain any arbitrary key-value pairs, Stormpath recommends that any email macro references to customData keys use the ``!`` quiet reference. Applications should also use the quiet reference because there are possible cases where the templating engine might not have access to an Application resource.
\ No newline at end of file
+Since customData can contain any arbitrary key-value pairs, Stormpath recommends that any email macro references to customData keys use the ``!`` quiet reference. Applications should also use the quiet reference because there are possible cases where the templating engine might not have access to an Application resource.
```

Comment:
Idk why Github sees this as a change. I didn't change anything here! :stuck_out_tongue_winking_eye: